woodwork/tests/logical_types/test_logical_types.py

```diff
@@ -659,3 +664,42 @@ def test_null_invalid_postal_code_numeric():
     expected = pd.DataFrame({"data": data})
     df.ww.init(logical_types=types, null_invalid_values=True)
     pd.testing.assert_frame_equal(df, expected)
+
+
+@pytest.mark.parametrize("null_type", [None, pd.NaT, np.nan, "null", "N/A", "mix", 5])
+@pytest.mark.parametrize("data_type", [int, float])
+def test_integer_nullable(data_type, null_type):
+    nullable_nums = pd.DataFrame(
+        np.random.choice([data_type(i) for i in range(10)], 100), columns=["num_nulls"]
```

Comment:
Always iffy about introducing randomness into unit tests....I think you're probably better off just building a dataframe with everything you want right off the bat.